### DISCUSSION TITLE
How to disable options of Select, Autocomplete and Combobox in vuetify 3

### DISCUSSION THREAD

**Discussion by hbgl** [vuetifyjs/vuetify#16816] [Q&A] [answered]
For my use case I need to disable some options in a Autocomplete component. I know this was possible in vuetify 2 but in version 3 the `disabled` property in the option object no longer works (see issue https://github.com/vuetifyjs/vuetify/issues/16792). Is there any workaround?

**Comment by hbgl** [comment, marked as answer]
I found a solution.

This does not work:
```tsx
<v-autocomplete
  label="Foo"
  :items="[
    { title: 'A', value: 1, disabled: true },
    { title: 'B', value: 2 },
    { title: 'C', value: 3 },
  ]"
 />
```

This works:

```tsx
<v-autocomplete
  label="Foo"
  item-props="props"
  :items="[
    { title: 'A', value: 1, props: { disabled: true } },
    { title: 'B', value: 2 },
    { title: 'C', value: 3 },
  ]"
/>
```

**Reply by Fedorov1977** [reply]
Which version of vuetify are you using? I'm on 3.5.15 and I can't get this to work.
In some earlier version of vuetify3 it was enough to just set "props: { disabled: true }" to disable it. But even if I add "item-props="props"" on the autocomplete it still does not disable the item.

UPDATE 1:
I see now that my issue was partially because "props" did not exist on the items in the array. If I add f.e "props:{disabled:true}" when the component renders then the items are disabled. 
However, my big issue is that I need to change some of the items to disabled after the component has renderd based on some other input from the user. I can see that the values for "disabled" changes correctly on each item but the actual item still does not get disabled (or enabled) if the change is done after the component has rendered the first time.

So it's sort of some "update issue" within the listitem component or something.

UPDATE 2:
Ok, problem solved. The problem was that I was binding my autocomplete to a computed. Then, I changed the disabled-prop on the items in the computed. This must have worked in vue2 but not in vue3 (or maybe it worked in some earlier version of vue3 because I'm sure I've tested this).

Anyways, now I bind my autocomplete to an array that is declared in the data-property. Now it works fine to change state of the props.disabled.
